Sources/IndexStoreDB/IndexStoreDB.swift

```diff
@@ -312,6 +312,41 @@ public final class IndexStoreDB {
     return result
   }
 
+  public func symbolProvider(for sourceFilePath: String) -> SymbolProviderKind? {
```

Comment:
I removed this function recently and it looks to me like this PR is accidentally adding it back. Could you rebase your branch to just contain your commits?

Reply:
oops, fixed

Reply:
would squashing the commits from this PR work?

Reply:
Yes, squashing is encouraged.